prototype/src/main/webapp/meeting-history.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>GoMeet Meeting History</title>
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 style="text-align:center;">GoMeet Meeting History</h1>
+    <ul>
```

Comment:
This `ul` element is a little weird - normally `ul` should only have `li` elements as children. If you like the 'indented' look of this, I would replace this with a `div` and add a `margin-left: 40px` or similar style.

Reply:
Done!